packages/amplify-category-auth/src/index.js

```diff
@@ -110,6 +110,7 @@ async function externalAuthEnable(context, externalCategory, resourceName, requi
     ? Object.assign(defaults.functionMap[requirements.authSelections](currentAuthName), currentAuthParams, immutables, requirements)
     : Object.assign(defaults.functionMap[requirements.authSelections](currentAuthName), requirements, {
         resourceName: `cognito${sharedId}`,
```

Comment:
The `sharedId` here can likely be updated to `defaults.sharedId` as well.

Reply:
Yeah I guess we can to maintain consistency and redundant uuid() code from different files.